arkane/ess/factory.py

```diff
@@ -113,10 +113,30 @@ def __init__(self, fullpath: str):
         backward compatible with current input files that use ``Log`` calls. Added a .ess attribute that stores
         the specific ESS Adapter, along with shortcut methods to call the respective Adapter method.
         """
-        self.path = fullpath
+        self._path = fullpath
```

Comment:
can you squash this commit into the larger factory commit?

Reply:
I just squashed your commit into the larger one